app/assets/stylesheets/basscss/_color-base.scss

```diff
@@ -16,3 +16,5 @@ hr {
   border-bottom-width: $border-width;
   border-bottom-color: $gray-6;
 }
+
+.muted { opacity: .5 }
```

Comment:
`.5` should be written with a leading zero as `0.5`<br>Declaration should be terminated by a semicolon